Bottom: 87328a64845ac5a419478eaf79ed92b1d05a2bca
Top:    184aac38cd8ef619183dcb3764d078ef599773db
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-30 11:52:57 +0800

Refresh of autofs-5.0.7-probe-each-nfs-version-in-turn-for-singleton-mounts.patch

---

diff --git a/modules/mount_nfs.c b/modules/mount_nfs.c
index 28ffcfc..c6400f0 100644
--- a/modules/mount_nfs.c
+++ b/modules/mount_nfs.c
@@ -180,10 +180,8 @@ int mount_mount(struct autofs_point *ap, const char *root, const char *name, int
 	 * We can't probe protocol rdma so leave it to mount.nfs(8)
 	 * and and suffer the delay if a server isn't available.
 	 */
-	if (rdma) {
-		singleton = 0;
+	if (rdma)
 		goto dont_probe;
-	}
 
 	/*
 	 * If this is a singleton mount, and NFSv4 only hasn't been asked
